ci/conda_recipe/condaversion.patch

```diff
@@ -0,0 +1,14 @@
+diff --git setup.py setup.py
+index 8af8b6d..4e4f9d2 100644
+--- setup.py
++++ setup.py
+@@ -57,6 +57,9 @@
+ import versioneer
+ __version__ = versioneer.get_version()
+ 
++# For conda builds...
++with open("__conda_version__.txt", "w") as f:
```

Comment:
This is not necessary (and needs to be removed). Runtime determination of the version can be done with:

```
{% set data = load_setuptools() %}
package:
   name: my_package
   version: {{ data.get('version') }}
```